lms/static/sass/multicourse/_dashboard.scss

```diff
@@ -254,6 +254,26 @@
         .course-container{
           border: 1px solid $border-color-l4;
           border-radius: 3px;
+          .xseries-container{
+            margin: ($baseline*0.4) ($baseline/5) 0 ($baseline*0.4);
+
+            .xseries-icon{
+                @include float(left);
+                display: inline;
+
+                @include margin-right($baseline*0.4);
+                background: url('#{$static-path}/images/icon-sm-xseries-black.png') no-repeat;
+                background-color: transparent;
+
+                width: ($baseline*1.1);
+                height: ($baseline*1.1);
+            }
+
+            p{
+              padding-top: ($baseline/5);
+              font-size: 14px;
```

Comment:
is this is copy, use the correct variables from _typography.scss. (We have extends for titles, subtitles, icons, and body copy. This is probably just for the label / body copy so I would use one of those.)